### DISCUSSION TITLE
Got error `("Connection broken: FileExistsError(17, 'File exists')", FileExistsError(17, 'File exists'))`

### DISCUSSION THREAD

**Discussion by hongbo-miao** [renovatebot/renovate#32463] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

Since a week ago, near every Poetry related pull requests from Renovate at https://github.com/hongbo-miao/hongbomiao.com/pulls, I got error

![image](https://github.com/user-attachments/assets/f9f57898-964e-49cd-9de7-d10a8671fb86)

If I check "If you want to rebase/retry this PR, check this box", the second time re-run most time will succeed.

Any ideas? Thank you! 

### Logs (if relevant)

Hmm I cannot check logs, when I click 
![image](https://github.com/user-attachments/assets/4bd13a81-771c-49a9-adbb-0bc9f236aee3)

it shows
![image](https://github.com/user-attachments/assets/e1bbb6c8-6b5f-4414-9a3e-957697370094)

**Comment by rarkins** [comment, marked as answer]
Can you determine which of these it is?
1. It was broken but now it's fixed, or
2. It's always broken first time, gets fixed second time, or
3. It's "randomly" broken?

**Reply by hongbo-miao** [reply]
Thanks @rarkins ! It was broken since a week ago, and ~90% time failed.
But after you commented, I haven't seen any failures 🤔
I will follow up if I see it again, thanks!
